cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -57,29 +59,115 @@
             </div>
         </div>
     </fieldset>
+    {% endif %}
 
     <fieldset class="o-form-input-group">
         <div class="form-l_col
                     form-l_col-1
                     form-l_col__flush-bottom">
+            {% if value.intro_text %}
+            <legend class="h3">
+                {{ value.intro_text }}
+            </legend>
+            <label for="comment">
+                <span class="h4">{{ value.question_text }}</span>
+            </label>
+            {% endif %}
+
+            {% if value.was_it_helpful_text %}
             <label for="comment">
                 <em>Optional</em> <span class="h4">Additional comment</span>
             </label>
+            {% endif %}
             <p class="u-mb10"><small><b>Note:</b> Do not include sensitive information like your name, contact information, account number, or social security number in this field.</small></p>
         </div>
         <div class="form-l_col
                     form-l_col-1">
-            <textarea id="comment" name="comment" rows="2">
+            <textarea id="comment" name="comment" rows="6">
                 {{- form.comment.value() | default('', true) -}}
             </textarea>
         </div>
+    </fieldset>
+
+        {% if value.optional_intro %}
+    <fieldset class="o-form-input-group">
         <div class="form-l_col
-                    form-l_col-1
-                    form-l_col__flush-bottom">
-            <button class="btn" type="submit">{{ value.button_text }}</button>
+                    form-l_col-1">
+            <h3>
```

Comment:
It might make sense to wrap these in `if` statements so they won't be included in the template if not provided.